docs/tutorial.md

```diff
@@ -6,6 +6,18 @@ title: Tutorial
 # Your first Aragon app
 In this guide, we will walk you through creating your first Aragon app using [aragonOS](os-ref.md), [aragon.js](js-ref.md), [Aragon UI](ui-intro.md) and [Aragon CLI](https://github.com/aragon/aragon-cli).
 
+# Quickstart
+
+```
+npm i -g @aragon/cli
+aragon init foo.aragonpm.eth react
+cd foo
+npm install
+npm start
```

Comment:
I would substitute `npm start` with `aragon run` to be consistent with the rest of the documentation.